hledger-web/hledger-web.m4.md

```diff
@@ -72,6 +72,10 @@ as shown in the synopsis above.
 `--port=PORT`
 : listen on this TCP port (default: 5000)
 
+`--socket=SOCKETFILE`
+: use a socket file to listen for requests instead of a TCP socket. Implies 
+`--serve`. It can only be used on systems that are capable of this type of socket.
```

Comment:
Let's be more explicit:  "...capable of this type of socket (eg, not Windows)." ?

Reply:
I just wanted to confirm it, and it seems like Windows *might* be able to provide `AF_UNIX` sockets now [1]. I'd say that explicitly calling windows out might be wrong, but I reworded it now so that the importance of the operating system becomes a bit more clear here.

[1] https://devblogs.microsoft.com/commandline/af_unix-comes-to-windows/